src/uct/api/uct.h

```diff
@@ -1578,6 +1741,23 @@ ucs_status_t uct_iface_reject(uct_iface_h iface,
 ucs_status_t uct_ep_create(const uct_ep_params_t *params, uct_ep_h *ep_p);
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Initiate a synchronized disconnection of an endpoint connected to a
```

Comment:
need to document:
- do we need to call uct_ep_destroy after completed?
- what happens if we try to communicate after disconnect?